matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerNetworkFactory.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.matpower.converter;
+
+import com.powsybl.commons.datasource.ResourceDataSource;
+import com.powsybl.commons.datasource.ResourceSet;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.NetworkFactory;
+
+/**
+ * @author Christian Biasuzzi <christian.biasuzzi@techrain.eu>
+ */
+public final class MatpowerNetworkFactory {
```

Comment:
What is the license of the different cases (i.e. compatible with ours)?

Reply:
test cases are the same tests as the ones used in ieee-cdf module (cdf files converted to the matpower format by a MATLAB tool cdf2mpc), so their licenses should be OK, I suppose ...